mailpoet/lib/Automation/Engine/Engine.php

```diff
@@ -73,6 +75,8 @@ private function registerApiRoutes(): void {
       $api->registerGetRoute('workflows', WorkflowsGetEndpoint::class);
       $api->registerPutRoute('workflows/(?P<id>\d+)', WorkflowsPutEndpoint::class);
       $api->registerPostRoute('workflows/create-from-template', WorkflowsCreateFromTemplateEndpoint::class);
+      $api->registerPostRoute('workflows/(?P<id>\d+)/duplicate', WorkflowsDuplicateEndpoint::class);
+      $api->registerDeleteRoute('workflows/(?P<id>\d+)/delete', WorkflowsDeleteEndpoint::class);
```

Comment:
The path should be only `workflows/(?P<id>\d+)`, I hope it works (that it doesn't override the POST, for instance).